rpms/SPECS/5.0.0/wazuh-manager-5.0.0.spec

```diff
@@ -480,23 +490,15 @@ fi
 
 # If the package is been uninstalled
 if [ $1 = 0 ];then
-  # Remove the ossecr user if it exists
-  if id -u ossecr > /dev/null 2>&1; then
-    userdel ossecr >/dev/null 2>&1
-  fi
-  # Remove the ossecm user if it exists
-  if id -u ossecm > /dev/null 2>&1; then
-    userdel ossecm >/dev/null 2>&1
-  fi
-  # Remove the ossec user if it exists
-  if id -u ossec > /dev/null 2>&1; then
-    userdel ossec >/dev/null 2>&1
+  # Remove the wazuh user if it exists
+  if id -u wazuh > /dev/null 2>&1; then
+    userdel wazuh >/dev/null 2>&1
   fi
-  # Remove the ossec group if it exists
-  if command -v getent > /dev/null 2>&1 && getent group ossec > /dev/null 2>&1; then
-    groupdel ossec >/dev/null 2>&1
-  elif id -g ossec > /dev/null 2>&1; then
-    groupdel ossec >/dev/null 2>&1
+  # Remove the wazuh group if it exists
+  if command -v getent > /dev/null 2>&1 && getent group wazuh > /dev/null 2>&1; then
+    groupdel wazuh >/dev/null 2>&1
+  elif id -g wazuh > /dev/null 2>&1; then
+    groupdel wazuh >/dev/null 2>&1
```

Comment:
Why double delete?, only want an explanation 😃

Reply:
In case the group cannot be obtained using the first command `getent group wazuh` it will try to check it with `id -g wazuh` and delete it if exists.